documentation/installation/manual_cli.md

```diff
@@ -0,0 +1,99 @@
+---
+id: manual_cli
+title: Installation, Linux command-line
+description: "Manual installation steps using the Linux command-line"
+---
+
+This guide will help you install HACS using the Linux command-line on your Home Assistant server.
+
+### Step 1 - Get the URL for the latest HACS release
+
+In your browser, visit the [HACS latest release page](https://github.com/custom-components/hacs/releases/latest).
```

Comment:
For a fully CLI installation, you should not need to go there.
There should be a way to get that URL by using CLI only.